packages/tab/index.tsx

```diff
@@ -0,0 +1,241 @@
+// The MIT License
+//
+// Copyright (c) 2018 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as React from 'react';
+import * as classnames from 'classnames';
+import TabIndicator from '@material/react-tab-indicator';
+// No mdc .d.ts files
+// @ts-ignore
+import {MDCTabFoundation} from '@material/tab/dist/mdc.tab';
+import TabRipple, {TabRippleProps} from './TabRipple';
+
+export {
+  TabRipple,
+  Tab,
+  TabRippleProps,
+};
+
+export interface TabProps extends React.HTMLProps<HTMLButtonElement> {
+  active?: boolean;
+  className?: string;
```

Comment:
Do you need to explicitly type `className` here, if you're extening `HTMLProps`?